variables.tf

```
@@ -14,20 +19,127 @@ variable "subnet_id" {
+  type        = string
+}
+
+variable "db_password" {
+  description = "The master password for the database user."
+  type        = string
+}
+
+variable "db_port" {
+  default     = 27017
+  description = "The port to use for the DocumentDB cluster."
+  type        = number
+}
+
+variable "db_username" {
+  description = "The master username for the database user."
+  type        = string
+}
+
+variable "ec2_trusted_ingress_cidr_blocks" {
+  default     = []
+  description = "The CIDR blocks to allow access to the EC2 instance."
+  type        = list(string)
+}
+
+variable "kevsync_lambda_cloudwatch_logs_retention_in_days" {
+  default     = 90
+  description = "The number of days to retain CloudWatch logs for the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = number
+}
+
+variable "kevsync_lambda_config_ssm_key" {
+  default     = "/cyhy-kevsync/config"
+  description = "The SSM key that contains the configuration to use for the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "kevsync_lambda_description" {
+  default     = "Syncs KEV data to the database in the Cyber Hygiene account."
+  description = "The description to associate with the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "kevsync_lambda_env_variables" {
+  default     = {}
+  description = "The environment variables to set for the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = map(string)
+}
+
+variable "kevsync_lambda_handler" {
+  default     = "lambda_handler.handler"
+  description = "The handler to use for the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "kevsync_lambda_name" {
+  default     = "cyhy-kevsync"
+  description = "The name to assign the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "kevsync_lambda_runtime" {
+  default     = "python3.12"
+  description = "The runtime to use for the Lambda function that syncs KEV data to the database in the Cyber Hygiene account."
+  type        = string
+}
+
+variable "kevsync_lambda_s3_key" {
+  default     = "cyhy-kevsync-lambda.zip"
+  description = "The key of the cyhy-kevsync Lambda deployment package in the S3 bucket."
+  type        = string
+}
+
+variable "kevsync_lambda_schedule" {
+  default     = "cron(0 6 * * ? *)"
+  description = "The EventBridge expression that represents when to run the Lambda function that syncs KEV data to the database in the Cyber Hygiene account.  The default value indicates that the Lambda will run every day at 6:00 AM UTC.  For details on EventBridge expression syntax, refer to <https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-scheduled-rule-pattern.html>"
```

Try using raw HTML here (I'm not sure if that will work):
```suggestion
  description = "The EventBridge expression that represents when to run the Lambda function that syncs KEV data to the database in the Cyber Hygiene account.  The default value indicates that the Lambda will run every day at 6:00 AM UTC.  See <a href="https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-scheduled-rule-pattern.html">here</a> for details on EventBridge expression syntax."
```

Even if that doesn't work you still need the period at the end of the sentence containing the link.

---

Our markdownlint rules currently prohibit inline HTML except for `<h1>` and `<img>` tags: https://github.com/cisagov/cyhy-tf-root/blob/db17957ed627dc9fe998cadc2cb7971a405222b0/.mdl_config.yaml#L35-L40

I don't think we want to allow `<a>` tags just for this, do you?

---

No, you are correct.

---

Yes, that is exactly why.  I am pretty sure I have done that elsewhere in the past, per the recommendation [here](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md).

---

Are you good with this modified suggestion?

```suggestion
  description = "The EventBridge expression that represents when to run the Lambda function that syncs KEV data to the database in the Cyber Hygiene account.  The default value indicates that the Lambda will run every day at 6:00 AM UTC.  See <https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-scheduled-rule-pattern.html> for details on EventBridge expression syntax."
```

---

Please see https://github.com/cisagov/cyhy-tf-root/pull/12/commits/a3d6f733bf02702c6b00fe63e97068f0d9327f05.